readme.rst

```diff
@@ -3,21 +3,72 @@
 Documentation starter pack
 ==========================
 
-The Documentation starter pack contains tools and configuration options for
-building a documentation site with Sphinx. Its main features include:
+*A pre-configured repository to build and publish documentation with Sphinx.*
 
-* bundled Sphinx theme, configuration, and extensions
-* build checks: links, spelling, inclusive language
+The Documentation starter pack includes:
 
-One strong point of the starter pack is its support for customisation that is
-layered on top of a core configuration - this applies to all the above
-features. There are also many other goodies included to enrich your doc set.
+* a bundled Sphinx_ theme, configuration, and extensions
+* support for both reStructuredText (reST) and Markdown
+* build checks for links, spelling, and inclusive language
+* customisation support layered above a core configuration
+
+Quickstart guide
+----------------
+
+An initial set of documentation can be built directly from a clone of this
+repository.
+
+First, clone this repository to a local directory, and change to this
+directory:
+
+.. code-block:: sh
+
+   git clone git@github.com:canonical/sphinx-docs-starter-pack alpha-docs
+   cd alpha-docs
+
+Now build the documentation with the following command. This will create a virtual
+environment, install the software dependencies, and build the documentation:
+
+.. code-block:: sh
+
+   make run
+
+Keep this session running to rebuild the documentation automatically whenever a
+file is saved, and open http://127.0.0.1:8000 in a web browser to see the
```

Comment:
```suggestion
file is saved, and open ``http://127.0.0.1:8000`` in a web browser to see the
```

Reply:
If you want to have this rendered as a link, you'll need to use the trick we use for, for example, `sphinx-design` at the bottom of the page.